presto-github/src/test/java/com/facebook/presto/github/GithubApiCredentials.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.github;
+
+public final class GithubApiCredentials
+{
+    public static final long ID = 25751824L;
+    public static final String NAME = "presto-github-connector";
+    public static final String TOKEN = "cdaa477e92005d2308469a9430c127a08dc26ea2";
```

Comment:
Make sure this token has very restricted access (aka just read). Delete it if it doesn't and generate another.